### DISCUSSION TITLE
A way to dispatch right after SUCCESS action

### DISCUSSION THREAD

**Discussion by BigBadAlien** [klis87/redux-saga-requests#525] [Q&A]
> onSuccess interceptor is fired after a successful response is received from the server, but before success action is dispatched

Do we have any way to dispatch other actions right after success action is dispatched? It's possible if resolve action as promise or use saga, but do we have any way to do it right inside the action's configuration?

**Comment by klis87** [comment]
Could you please provide an example what you are triung to achieve? Sry to late response btw, I did not see any alert for discussions
